js/anims-missiles.js

```diff
@@ -13,54 +16,71 @@ function dropMissile(animationTime) {
 
     $missile.animate({
         top: "+=550"
-    }, animationTime, "linear", function () {
-        missileHitsCity()
-        $(this).remove()
+    }, animationTime, "linear", function() {
+        missileHitsCity($(this))
     })
 }
 
+/**
+ * Function call drops missile every 500ms
+ * animationTime is variable based on users score
+ */
 function repeatAnims() {
     setInterval(function() {
-        dropMissile(2000)
+        var animationTime = animationChangeSpeed()
+        dropMissile(animationTime)
     }, 500)
 }
 
-function missileHitsCity() {
+/**
+ * Function decrements game score and increments death toll and renders the scores on the page when a city is hit
+ * @param that Had to pass the $(this) value into missileHitsCity() as 'that' because 'this' is a reserved word and therefore cannot pass 'this' as a parameter
+ */
+function missileHitsCity(that) {
```

Comment:
that isnt a great variable name. its the missile, why not use $missile

Reply:
Fixed in 2cb838a